CoastalResilienceNetwork/tnc-network-site
# Add map markers

## Overview

Adds markers for subregions for the active region. See commit messages for details.

Connects #13 

## Demo

![tnc1](https://user-images.githubusercontent.com/1042475/32466257-4b822074-c314-11e7-8a5b-b2389ae6d7bc.gif)

## Testing Instructions

- Select a subregion. Verify that markers are added only for that subregion.
- Click a marker. Verify a popup is opened, and that the marker (real a copy of the marker) is placed above the popup window and acts as a popup tip.
- Click the "Open map" button in the popup and ensure that a new tab is opened to the selected region.
- Unactivate the region. Verify that the markers are removed.
